apps/chef-battle/src/lib/production/checkpoint.ts

```
@@ -0,0 +1,745 @@
+		}
+		case 'kitchenShowdownStart':
+			if (event.bonusBankAtomicUnits !== next.bonusBankAtomicUnits)
+				replayError('INVALID_BOOK', 'kitchenShowdownStart Bank must preserve reducer continuity');
+			const openingQueue = (['italian', 'french', 'chinese'] as const)
+				.filter((chef) => event.meters[chef] === 100)
+				.map((chef) => ({
+					id: `${event.roundId}-service-01-${chef}`,
+					chef,
+					readySequence: event.sequence,
+					perfectServeUnits: 0,
+				}));
```

Biome (2.5.8)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)